### PR TITLE
#657: Create properties file for running against Confluent Cloud

### PR DESCRIPTION
**Pull request by ybyzek**
### Description

https://github.com/confluentinc/kafka-tutorials/issues/657

### Staging Docs

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-657

View any/all tutorials that run kafka or kstreams, for example: http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-657/create-stateful-aggregation-count/kstreams.html#run-your-app-to-confluent-cloud

### CODE REVIEW

File: _includes/shared/markup/ccloud/try-ccloud-properties.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Use the promo code `CC100KTS` to receive an additional $100 free usage (https://www.confluent.io/confluent-cloud-promo-disclaimer[details]).
+
+Next, from the Confluent Cloud UI, click on `Tools & client config` to get the cluster-specific configurations, e.g. Kafka cluster bootstrap servers and credentials, Confluent Cloud Schema Registry and credentials, etc., and set the appropriate parameters in your client application.
+In the case of this tutorial, add the following properties to the client application's input properties file, substituting all curly braces with your Confluent Cloud values.
+
++++++
+<pre class="snippet"><code class="text">{% include_raw shared/code/ccloud/properties.txt %}</code></pre>
++++++
+
+Now you're all set to your run application locally while your Kafka topics and stream processing is backed to your Confluent Cloud instance.
```
Comment (rspurgeon) [inline _includes/shared/markup/ccloud/try-ccloud-properties.adoc:13]: FYC (I felt the "kafka topics and stream processing" was slightly disjointed):

```suggestion
Now you're all set to your run streaming application locally backed by a Kafka cluster fully managed by Confluent Cloud.
```

**Review by rspurgeon — APPROVED** [review]
LGTM w/ one optional copy comment

File: _includes/shared/markup/ccloud/try-ccloud-properties.adoc (hunk 1 above)
Comment (rspurgeon) [reply _includes/shared/markup/ccloud/try-ccloud-properties.adoc]: @ybyzek My edit is terrible grammatically on a second pass!

Comment (rspurgeon) [reply _includes/shared/markup/ccloud/try-ccloud-properties.adoc]: "run your" not "your run"

**Review by awalther28 — APPROVED** [review]
LGTM, I appreciate the contribution. It helps make that CCloud block more actionable.

**Review by bbejeck — APPROVED** [review]
Thanks for the PR @ybyzek, LGTM!

This is a great addition that makes the `take it to cloud` more tangible.

**Comment by ybyzek** [comment]
Thanks all!  Merging now...

**Pull request merged**